exo-test/bin/setup

```diff
@@ -1,2 +1,12 @@
 #!/usr/bin/env bash
-yarn install
+
+# check if yarn is installed
+command -V yarn >/dev/null 2>&1
```

Comment:
Are you just testing for the existence of `yarn` here? This isn't posix compliant. It's better to just use `which`.

```shell
if which yarn; then
  yarn install
else
  echo "ERROR: yarn is not installed."
  echo "       visit https://yarnpkg.com/en/docs/install to see installation instructions based on your OS"
  exit 1
fi